api/spec/api_docs/definitions/resource.rb

```diff
@@ -80,14 +108,20 @@ def write_only_attributes
         self::REQUEST_ATTRIBUTES
       end
 
-      def make_request(callee, attributes)
-        attributes = ::Types::Hash.schema(attributes)
-        attributes = attributes.meta(required: required_create_attributes) if required_create_attributes.present? && callee == :create_request
-        attributes = attributes.meta(required: required_update_attributes) if required_update_attributes.present? && callee == :update_request
+      def build_value(contents, required_values)
```

Comment:
I could not think of a great name for this method, if you have a suggestion, that would be appreciated.

Reply:
I'm not sure what this method is doing. Can you explain what it's doing? Maybe the word you're looking for here is `hydrate`, which refers to the process of filling an object with data?

Reply:
I cleaned up the code a bit, so even though there are changes, it should be more straightforward to understand.